gnomad/resources/grch38/gnomad.py

```diff
@@ -470,11 +495,19 @@ def gnomad_gks(
     high_level_version = f"v{version.split('.')[0]}"
 
     # Read coverage statistics if requested
-    if high_level_version == "v3":
-        coverage_version = "3.0.1"
+    coverage_version_3_0_1 = "3.0.1"  # v4 genomes coverage
+    coverage_version_4_0 = "4.0"  # v4 exomes coverage
+
+    # In v4, exomes have coverage in v4 coverage table,
+    #   genomes have coverage in v3 coverage table.
+    if data_type == "genomes":
+        coverage_version = coverage_version_3_0_1
     else:
+        coverage_version = coverage_version_4_0
+
+    if high_level_version != "v4":
```

Comment:
Just occurred to me - would make more sense to move this to the start of the function (and accompanying high_level_version logic) before reading in any tables. 

@klaricch your other PR has already been merged and we can leave it up to the other functions to fail for bad keys, yes ? We don't want to check here that like 'version' is in the list of valid versions or anything, yeah ?

Reply:
I will move the high_level_version stuff up above the first table read